readme.md

```diff
@@ -1,30 +1,51 @@
 # Prototype Playground
 What is it? This is a base environment, with the minimum amount of setup and dependencies to start building web prototypes.
 
+You'll be setting up Node, a Javascript environment that lets you code on your local machine. With Node, you can use NPM and Gulp. NPM is a package manager that lets you download useful tools to help you code. Gulp is a task runner that automates whatever you tell it to. It's commonly used to check for errors, compile your code, and auto refresh your browser.
+
 # Getting started
-Clone this repository, then:
+1. Install [Atom](https://atom.io/) or another text editor of your choice
+2. Download XCode from the App Store
+3. Open the Terminal app
+4. Install Homebrew. This makes it [easier to install Node](http://blog.teamtreehouse.com/install-node-js-npm-mac ).
+    1. In Terminal, type ``ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" ``
+    2. You’ll see messages in the Terminal explaining what you need to do to complete the installation process.
+5. Install Node and NPM
+    1. Once Homebrew is installed, type ```brew install node```
+    2. To make sure Node is installed, type `node -v`. You should see the version that's installed, ex:  `v0.10.31`.
+    3. Make sure NPM is installed by typing `npm -v`
```

Comment:
This is pretty complicated for a beginner. It's much easier to install node by downloading it from nodejs.org.